src/WebSocket.cpp

```diff
@@ -548,6 +548,30 @@ namespace WebSocket
             }
             ClientServerBase::~ClientServerBase() {} ///< for vtable
 
+            bool ClientServerBase::checkHeaders(QString *what) const
+            {
+                const auto Fail = [what](const QString &_what) {
+                    if (what != nullptr)
+                        *what = _what;
+                    return false;
+                    };
```

Comment:
Mind flushing the lambda close brace to the left?

Reply:
done